### ISSUE TITLE
Better indicate external link behavior

### ISSUE THREAD

**Issue by aviannamiller** [omeka/omeka-s#2531]
Links that open in new tabs are currently indicated by external link icons. It is best practice to indicate if a link opens in a new tab, so we should be supplying an additional label accessible to screen readers ("Opens in new tab"). A tooltip that reads the same would also be appropriate, ultimately.

This should mainly involve targeting all links associated with `.o-icon-external` icons, but there are other cases, such as "View" buttons within page actions, which do not currently indicate external behavior visually, either.

**Comment by kimisgold** [comment]
The branch `a11y/external-link-icons` is ready for initial testing.
